src/Enhavo/Bundle/NavigationBundle/Renderer/NodeRenderer.php

```diff
@@ -12,10 +12,12 @@
 use Enhavo\Bundle\NavigationBundle\Exception\RenderException;
 use Enhavo\Bundle\NavigationBundle\Model\NodeInterface;
 use Enhavo\Bundle\NavigationBundle\NavItem\NavItemManager;
+use Symfony\Component\DependencyInjection\ContainerAwareTrait;
 use Twig\Environment;
 
 class NodeRenderer
 {
+    use ContainerAwareTrait;
```

Comment:
Why we need a `ContainerAwareTrait` here? I can't find any access to it in the `NodeRenderer.php`.